modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.tsx

```diff
@@ -27,16 +27,25 @@ function Actions({
 	itemId: string | number;
 }) {
 	const {
+		allItemsSelectedActive,
 		executeAsyncItemAction,
 		highlightItems,
 		inlineEditingSettings,
 		loadData,
 		onActionDropdownItemClick,
 		openModal,
 		openSidePanel,
+		selectedItemsValue,
 		toggleItemInlineEdit,
 	}: IFrontendDataSetContext = useContext(FrontendDataSetContext);
 
+	const isRowChecked =
+		allItemsSelectedActive ||
+		(selectedItemsValue &&
+			!!selectedItemsValue.find(
+				(element: any) => String(element) === String(itemId)
+			));
+
```

Comment:
I think we can simplify this:
- Instead of `selectedItemsValue && ...`, possibly use optional chaining with `?`
- Instead of `.find`, possibly use `.some` since it already returns a boolean
- Instead of `element`, possibly rename it to `selectedItemValue` to relate more as an item of `selectedItemsValue`

```
const isRowChecked =
	allItemsSelectedActive ||
		selectedItemsValue?.some(
			(selectedItemValue) => String(selectedItemValue) === String(itemId)
		);
```